### PR TITLE
[stable/postgresql] Add pg_hba.conf option.

### PR DESCRIPTION
**Pull request by desaintmartin**
Allow to set a pg_hba.conf file for postgres.
ping @databus23

### CODE REVIEW

**Comment by desaintmartin** [comment]
Guys, as this feature is mandatory for my project, should I fork/extend it and deploy my own helm repo or may such changes be incorporated here?

File: stable/postgresql/templates/deployment.yaml [hunk 1]
```diff
@@ -56,6 +56,10 @@ spec:
           - -c
           - '{{ $key | snakecase }}={{ $value }}'
           {{- end }}
+          {{- if .Values.pgHbaConf }}
+          - -c
```
Comment (cpanato) [inline stable/postgresql/templates/deployment.yaml:60]: duplicate `-c`

**Review by cpanato — CHANGES_REQUESTED** [review]

File: stable/postgresql/templates/deployment.yaml (hunk 1 above)
Comment (desaintmartin) [reply stable/postgresql/templates/deployment.yaml:60]: The diff is too narrow to show it properly, "-c" is for specifying a postgresql options from cli!

Comment (cpanato) [reply stable/postgresql/templates/deployment.yaml:60]: Ah, got it, read the documentation. Thanks!

**Comment by cpanato** [comment]
@desaintmartin you need to rebase this one as well

File: stable/postgresql/Chart.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 name: postgresql
-version: 0.15.1
+version: 0.15.2
```
Comment (davidkarlsen) [inline stable/postgresql/Chart.yaml]: new feature is added - make it 0.16.0 instead

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

File: stable/postgresql/Chart.yaml (hunk 2 above)
Comment (desaintmartin) [reply stable/postgresql/Chart.yaml]: Done.

**Pull request merged**